Hunting Queries/MultipleDataSources/PotentialMicrosoftSecurityServicesTampering.yaml

```diff
@@ -46,6 +49,31 @@ query: |
   | project TimeGenerated, Computer, Account, AccountDomain, ProcessName, ProcessNameFullPath = NewProcessName, EventID, Activity, CommandLine, EventSourceName, Type
   ),
   (
+  WindowsEvent
+  | where EventID == 4688 
+  | where (EventData  has_any (action) and EventData has_any (service1)) 
```

Comment:
- filter for has_any(includeProc) prior to the big and complex where clause driven from the CommandLine filter.